### ISSUE TITLE
Label is not associated with  for ‘Name for the block’ input field.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#134]
User Experience:

If label is not associated with form field than screen reader user will not be able to find out what kind of information is needed in the  fields and will face difficulties while navigating on the page.

 

Note: User credentials should NOT be included in the bug.
 

Repro-Steps

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any section details and select it.

7. Navigate to the Add button and select it.

8. Navigate to the various controls of the  "Add library" dialog box.

9. Open developer tool using F12 key and navigate to ‘Name for the block’ edit field.

10. Verify the issue.  

 

Actual Result:

Label is not associated with  for ‘Name for the block’ input field,  present in ‘Add to library’ popup, screen reader does not announce label.

 

Expected result:

Label should be associated with  for ‘Name for the block’ input field,  present in ‘Add to library’ popup. 

 

Note:

Similar issue exist with Chrome.

Similar issue exist throughout the application for same controls.

 

MAS Reference:

MAS 3.3.2 - Labels or Instructions (19)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
 

 

![image](https://user-images.githubusercontent.com/2320302/79025228-fad77d00-7b39-11ea-80dd-1da1e2fdc1ce.png)

**Comment by azaslonov** [comment]
Fixed.
